lib/chef/resource/locale.rb

```diff
@@ -113,7 +113,7 @@ def define_resource_requirements
 
           requirements.assert(:all_actions) do |a|
             # RHEL/CentOS type platforms don't have locale-gen
-            a.assertion { shell_out("locale-gen") }
+            a.assertion { shell_out("locale-gen -h") }
```

Comment:
When using `-h` on ubuntu, arch it still generates locale files - it does not seem to support any flags.

Instead, you can invoke `which("local-gen")` which will return `false` if the executable does not exist.

Reply:
I tested on Ubuntu and it worked. But i guess which would be a more generic solution. Will update PR.